src/main/java/net/mcreator/ui/minecraft/states/entity/JEntityDataList.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.ui.minecraft.states.entity;
+
+import net.mcreator.ui.MCreator;
+import net.mcreator.ui.component.util.*;
+import net.mcreator.ui.dialogs.MCreatorDialog;
+import net.mcreator.ui.help.IHelpContext;
+import net.mcreator.ui.init.L10N;
+import net.mcreator.ui.init.UIRES;
+import net.mcreator.ui.minecraft.JEntriesList;
+import net.mcreator.ui.minecraft.states.DefaultPropertyValue;
+import net.mcreator.ui.minecraft.states.PropertyData;
+import net.mcreator.ui.validation.Validator;
+import net.mcreator.ui.validation.component.VTextField;
+import net.mcreator.ui.validation.validators.JavaMemberNameValidator;
+import net.mcreator.ui.validation.validators.UniqueNameValidator;
+
+import javax.swing.*;
+import javax.swing.table.DefaultTableModel;
+import javax.swing.table.TableCellEditor;
+import java.awt.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class JEntityDataList extends JEntriesList {
+
+	private final DefaultTableModel entriesModel;
+	private final JTable entries;
+
+	public JEntityDataList(MCreator mcreator, IHelpContext gui) {
+		super(mcreator, new BorderLayout(), gui);
+		setOpaque(false);
+
+		entriesModel = new DefaultTableModel(new Object[] { L10N.t("elementgui.living_entity.entity_data_entries.name"),

```

Comment:
I would not use table but same system other entry lists use, eg bounding boxes, item properties,etc.

Reply:
You suggest using table here youselves earlier. Turns out to be worse?

Reply:
You are correct I asked this.

I checked the code and indeed it gets more complex. I think if we can make component align the same way as in other PRs using entry list in its pure form would be better.

Reply:
The way I showed a moment before the table suggestion would be fine? (In case you remember)

Reply:
Huh, it was a while. Do you have a link to a comment?

Reply:
I can find it in a minute.

Reply:
✔️ Fixed